tasks/configure-RedHat.yml

```diff
@@ -65,24 +81,10 @@
   when:
     - ansible_distribution_major_version | int >= 8
     - not localhost_cert.stat.exists
+
```

Comment:
Unrelated whitespace change.

Reply:
I would stick with this newline, it seems like this was a formatting error in the first place. If you insist, I will delete it 😛

Reply:
Ok :)